.github/workflows/pr-testing-with-test-project.yml

```
@@ -105,4 +105,8 @@ jobs:
       - if: needs.changes.outputs.should_test == 'true'
         name: Run PY acceptance tests
         run: docker compose -f ./microcks-setup/microcks-podman.yml --profile test-py up  --abort-on-container-exit --force-recreate
+        working-directory: ./packages/templates/clients/websocket/test
+      - if: needs.changes.outputs.should_test == 'true'
+        name: Run JAVA acceptance tests
+        run: docker compose -f ./microcks-setup/microcks-podman.yml --profile test-java up  --abort-on-container-exit --force-recreate
         working-directory: ./packages/templates/clients/websocket/test
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)